### PR TITLE
Feat: Added questions, results, rules page

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Rules page
Screenshot (310)
Question page
Screenshot (311)
Results page
Screenshot (312)

### CODE REVIEW

**Review by sakshamAK — APPROVED** [review]
Great work

File: CSS/commonstyles.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.margin-l {
```
Comment (srishti-maurya) [inline CSS/commonstyles.css:1]: maybe you can name it as `margin-lg` to convey the msg clearly

**Review by srishti-maurya — APPROVED** [review]
looks good

**Review by Varu98 — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**